announcements/1.12.5.md

```diff
@@ -0,0 +1,27 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.12.5.
+
+This is an out of order release to prepare for an upcoming Ubuntu kernel release on AWS. With the 5.13 kernel, which will be introduced on instances running Ubuntu 20.04 (focal) starting end of February, Anbox needs to be adopted to work with changes in the `/proc` file system and semantic changes to the mount system API. If you are running on Ubuntu with a kernel version greater or equal than 5.13 please ensure you're using the 1.12.5 images. Using older images will result in Anbox and the nested Android container failing to start.
```

Comment:
```suggestion
This release prepares for an upcoming Ubuntu kernel release on AWS. With the 5.13 kernel, which will be introduced on instances running Ubuntu 20.04 (focal) starting at the end of February, Anbox must be adopted to work with changes in the `/proc` file system and semantic changes to the mount system API. If you are running on Ubuntu with kernel version 5.13 or later, make sure you're using the 1.12.5 images. Using older images will result in Anbox and the nested Android container failing to start.
```

Reply:
Do we need the "out of order"? This is kinda misleading since it usually means "broken". ;)
If it is required, we could go for "unplanned" or "unscheduled", but I don't think it's really needed.